guide/voice/the-basics.md

```diff
@@ -0,0 +1,123 @@
+# The Basics
+
+Now that you have your dependencies installed, you can start using voice!
+
+## Joining voice channels
+
+Joining a voice channel is easy, here's a common example:
+
+```js
+client.on('message', async message => {
+	// Join the same voice channel of the author of the message
+	if (message.member.voice.channel) {
+		const connection = await message.member.voice.channel.join();
+	}
+});
+```
+
+If you try to join a channel the bot is already in, nothing will happen so we don't need to worry about running this method repeatedly.
+
+::: tip
+In the above example we accessed a `VoiceState` through `member.voice`. A voice state exists for members connected to a voice channel, and it tells you which channel they're connected to, if the member is speaking, and whether they're deafened or muted. Consult the documentation for a full description of what voice states can do!
+:::
+
+## Playing audio
+
+Playing audio is also simple. A `StreamDispatcher` is created for you to control and monitor the playback of your audio over the voice channel.
+
+A VoiceConnection can only ever have at most one dispatcher. If you try to play another stream while audio is already playing, the existing stream will be destroyed and the new audio will begin to play shortly after.
+
+```js
+// Create a dispatcher
+const dispatcher = connection.play('audio.mp3');
+
+dispatcher.on('start', () => {
+	console.log('audio.mp3 is now playing!');
+});
+
+dispatcher.on('finish', () => {
+	console.log('audio.mp3 is finished playing!');
+});
+
+// Always remember to handle errors appropriately!
+dispatcher.on('error', console.error);
+```
+
+To end the stream yourself, you can run:
+
+```js
+dispatcher.destroy();
+```
+
+You can also create the dispatcher with options. The following example will play a stream at 50% volume and with 3 passes (this is the number of times an audio packet is sent. Increasing this value will reduce the chance of packet loss, but will also increase bandwidth!)
+
+```js
+connection.play('audio.mp3', { volume: 0.5, passes: 3 });
+```
+
+### What sources of audio can I play?
+#### FFmpeg
+
+In the example shown above, you can play a file from its path, this is an example of using FFmpeg to play a file. You can also pass a `ReadableStream` (or path) of most media files, e.g. mp3, mkv, mp4, and this will be played. You can even play from URLs!
```

Comment:
I'd seperate the first sentence into two separate ones, turning "This is an example of using FFmpeg to play a file." into the second sentence, as it reads a bit oddly to me when attached as subordinate clause.